dao/searchDao.js

```diff
@@ -7,7 +7,45 @@ exports.getAll = function(filter, cb) {
     if(filter.class) {
         classFilter = ' AND class = ' + db.escape(filter.class);
     }
-    let sqlQuery = 'SELECT * FROM tbl_tickets WHERE reserver IS NULL' + classFilter + ';';
+
+    // filter to be used on sql query
+    let seatFilter = '';
+    let seatArray = null;
+    // still in string form after getting passed as a query parameter
+    // if there is something the seat object
+    if(filter.seat) {seatArray = filter.seat.split(',');}
```

Comment:
`if` is not a function; in new code, at least, please make sure there is a space between that keyword and the parenthesis surrounding its condition.

Reply:
Yes, I can do that in #7